azure-classic/azure-ts-appservice/package.json

```diff
@@ -5,7 +5,7 @@
         "@types/node": "^8.0.0"
     },
     "dependencies": {
-        "@pulumi/azure-nextgen": "^0.6.1",
```

Comment:
This seems wrong? Isn't this a native example?

Reply:
Or is this just git getting confused?

Reply:
Looks like git confusion, perhaps. The example in `azure-ts-appservice` looks like a classic example.